crates/core/component/ibc/src/component/packet.rs

```diff
@@ -161,7 +161,7 @@ pub trait SendPacketRead: StateRead {
         if packet.timeout_timestamp <= chain_ts.nanoseconds() {
             anyhow::bail!(
                 "timeout timestamp {} is less than the latest timestamp on the counterparty {}",
-                packet.timeout_height,
+                packet.timeout_timestamp,
                 chain_ts,
```

Comment:
can you change this to log `chain_ts.nanoseconds()` instead so the displayed types are the same?